x-pack/plugins/infra/public/components/asset_details/tabs/processes/processes.tsx

```diff
@@ -90,87 +94,106 @@ export const Processes = () => {
 
   return (
     <ProcessListContextProvider hostTerm={hostTerm} to={currentTimestamp}>
-      <SummaryTable
-        isLoading={loading}
-        processSummary={(!error ? response?.summary : null) ?? { total: 0 }}
-      />
-      <EuiSpacer size="m" />
-      <EuiText>
-        <h4>
-          {i18n.translate('xpack.infra.metrics.nodeDetails.processesHeader', {
-            defaultMessage: 'Top processes',
-          })}{' '}
-          <EuiIconTip
-            aria-label={i18n.translate(
-              'xpack.infra.metrics.nodeDetails.processesHeader.tooltipLabel',
+      <EuiFlexGroup direction="column" gutterSize="m">
```

Comment:
Changed to use flexbox instead of having `<EuiSpacer size="m" />` to separate each section